### ISSUE TITLE
Chip: missing keyboard navigation

### ISSUE THREAD

**Issue by tishoyanchev** [Infineon/infineon-design-system-stencil#2486]
### Detailed description of how to reproduce the problem

The keyboard navigation is missing 

### Upload a screenshot

NA

### Describe the development framework

Any

### Expected behavior

keyboard navigation to allow users to access and navigate the chip using space and enter keys
